runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -92,6 +99,28 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 	virtual bool preMonitorTableSlotRead(J9JavaVM *vm, j9object_t *srcAddress);	
 #endif	
 
+	/* New methods */
+	bool preObjectStoreInternal(J9VMThread *vmThread, J9Object *destObject, fj9object_t *destAddress, J9Object *value, bool isVolatile);
+	bool preObjectStoreInternal(J9VMThread *vmThread, J9Object **destAddress, J9Object *value, bool isVolatile);
+
+	/* New methods */
+	void rememberObjectToRescan(MM_EnvironmentBase *env, J9Object *object);
+
+	MMINLINE bool isSATBBarrierActive(MM_EnvironmentBase* env)
+	{
+		return _extensions->configuration->isSnapshotAtTheBeginningBarrierEnabled() &&
+				!_extensions->sATBBarrierRememberedSet->isGlobalFragmentIndexPreserved(env);
```

Comment:
the first part might not be necessary, since the second part would never be satisfied if SATB is not enabled

Reply:
fixed